### PR TITLE
[14][ADD] connector_prestashop_environment

### PR DESCRIPTION
**Pull request by florian-dacosta**
@flotho @hailangvn @jesusVMayor

### CODE REVIEW

File: connector_prestashop_environment/models/prestashop_backend.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class PrestashopBackend(models.Model):
+    _name = "prestashop.backend"
+    _inherit = ["prestashop.backend", "server.env.mixin"]
+
+    @property
+    def _server_env_fields(self):
+        base_fields = super()._server_env_fields
+        mail_fields = {
```
Comment (hailangvn) [inline connector_prestashop_environment/models/prestashop_backend.py]: Just curious, why is the name `mail_fields`?

**Review by hailangvn — APPROVED** [review]
LGTM. The PR title should be `[14.0][ADD] connector_prestashop_environment` so that we can search easily.

**Review by flotho — APPROVED** [review]
thanks LGTM

File: connector_prestashop_environment/models/prestashop_backend.py (hunk 1 above)
Comment (florian-dacosta) [reply connector_prestashop_environment/models/prestashop_backend.py]: Bad copy paste I guess ;)

**Comment by flotho** [comment]
could we expect a merge today ?

**Comment by flotho** [comment]
hi @OCA/connector-prestashop-maintainers  any chance to be merged ?

**Comment by bguillot** [comment]
Thanks:

/ocabot merge bump

**Comment by bguillot** [comment]
Thanks:

/ocabot merge nobump

**Pull request merged**